main.py

```diff
@@ -7,7 +7,7 @@
 
 logging.config.dictConfig(config.logging)
 
-uvicorn_args = {"port": 80}
+uvicorn_args = {"port": 8080}
```

Comment:
Our good old friend again ;-)

Reply:
I suggest we revert commit c395cbe. Using port 80 was never a good idea in the first place.

Reply:
So basically having macOS users suffer and not Linux users ;-) Ideally we would revert and have a mechanism to overwrite this setting with an .env file.

Reply:
I wouldn't call your use case suffering.

Anyway, with the planned Docker Compose support, you will get to easily run a usable server on your machine.

Alternatively, you can also do the following:
```
python3 -m venv venv
. venv/bin/activate
python -m pip install -e .

uvicorn --port 8080 --reload goosebit:app
```

Reply:
> So basically having macOS users suffer and not Linux users ;-) Ideally we would revert and have a mechanism to overwrite this setting with an .env file.

I'm using Arch, and this is still a problem by default...

Reply:
How about port `60053`?  Best port approximation for `GOOSE` ;)

Reply:
It is indeed a funny idea. However, IMHO there is nothing wrong with 8080, it is a common choice for the use case. In my experience, ports in the 60k range are rather uncommon.

Reply:
> It is indeed a funny idea. However, IMHO there is nothing wrong with 8080, it is a common choice for the use case. In my experience, ports in the 60k range are rather uncommon.

Fair enough.  Figure we might as well commit to the bit though.

My biggest concern is 8080 is usually pretty common for testing purposes, but not a big deal in production.

How about we just add it to the config file (no idea why its not in there), and then it can be passed there or as an ENV variable.

Reply:
Handled with #92